### DISCUSSION TITLE
creating new files - templates

### DISCUSSION THREAD

**Discussion by kal60** [gsantner/markor#1753] [Q&A] [answered]
(I'm currently exploring markor more and more - it's geat!)
running markor 2.9.0 on Android 12 on Samsung Galaxy Tab S7+

some questions concering creating new files:
1. would it be possible to keep the latest selected template file ? Currently it switches back to "empty file" every you create a new file. If e.g. you want to create a series of hugo-files, would be nice to keep the last selected template. Workaround would be using "empty file" and then using snippets
2. are the file templates hard coded or can you add your own template files (like snippets) ?

cheers.
Knut

**Comment by gsantner** [comment, marked as answer]
2 - With the next Markor update custom templates will be selectable. It will list the snippets at new file dialog with the Snippets system, which is already there in 2.9.0.

Am 7. Juli 2022 23:58:46 MESZ schrieb Knut Alboldt ***@***.***>:

**Reply by kal60** [reply]
Great :-D
